extensions/Datatype.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+trait Datatype {
```

Comment:
Please add the suffix Trait here (and in the filename)

Reply:
I'm not a friend of such suffix' but have changed it.